src/command/local/local.c

```diff
@@ -24,6 +24,11 @@ cmdLocal(int handleRead, int handleWrite)
 
     MEM_CONTEXT_TEMP_BEGIN()
     {
+        // Configure two retries for local commands
+        VariantList *retryInterval = varLstNew();
+        varLstAdd(retryInterval, varNewUInt64(0));
+        varLstAdd(retryInterval, varNewUInt64(15000));
```

Comment:
That's the idea. An immediate retry seems good because transient errors should be correctable immediately, and the 15 second delay is to get over any temporary network blips if the immediate retry did not work. The decision on time was pretty arbitrary -- mostly because I don't want a long delay when something is likely to fail anyway. We've gotten by without these retries for a long time so they are not very common.